tests/test_low_pass_filter.py

```diff
@@ -0,0 +1,13 @@
+# This script tests the low_pass_filter function
+
+import numpy as np
+import pytest
+from AudioFilters.low_pass_filter import low_pass_filter
+
+input_signal = np.genfromtxt('data/bark.csv', dtype = 'float32')
+
+def test_low_pass_filter_attenuates_frequencies_below_1000_cutoff():
+    expected_output = np.genfromtxt('data/lowpass/bark_highpass_1000Hz_24db.csv', dtype = 'float32')
```

Comment:
I think the name of the file should be `bark_lowpass_1000Hz_24db.csv` instead of `bark_highpass_1000Hz_24db.csv`

Reply:
Yes, I left the filename the same because it needs to be replaced with a completely different file.  I opened an issue for that.

Reply:
Filename is now changed.

Reply:
Hi Paul, just change description at the Readme file.